_includes/head.html

```diff
@@ -11,7 +11,7 @@
   <meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
   <link rel="icon" href="/favicon.ico" type="image/x-icon" />
 
-  <link rel="stylesheet" href="{{ site.url }}/assets/css/style.css">
```

Comment:
What would you say the `{{ site.url }}` is needed for? It seems to be the cause of the CSS issue that we observed (as the site.url is interpreted as http) and also makes it problematic to run the site on localhost (as the css file is in that case still fetched over the internet, meaning eventual changes would not be applied). It would also stop us from serving the site over multiple urls, e.g. an onion site.